core/tests/protractor.conf.js

```diff
@@ -77,6 +77,10 @@ var suites = {
       'protractor_desktop/extensions.js'
     ],
 
+    featureGating: [
+      'protractor/featureGatingFlow.js' // TODO
```

Comment:
What is the TODO for? If some work needs to be done after this PR gets merged, please file an issue, assign it to yourself, and mention the issue in the TODO comment like we have done in other places in the codebase.

Reply:
Ah nevermind, it's a reminder for myself that I forgot to delete -- deleted it in the latest commit.